ide/api.debugger/src/org/netbeans/api/debugger/DebuggerManager.java

```diff
@@ -315,7 +315,7 @@ public DebuggerEngine[] startDebugging (DebuggerInfo info) {
             }
             
             // init DebuggerEngines
-            ArrayList engineProviders = new ArrayList ();
+            List engineProviders = new ArrayList<>();
```

Comment:
`List<Object>` on the left. Since that is unfortunately the common type the way this list is used.

Reply:
done..